lib/rack/utf8_sanitizer.rb

```diff
@@ -76,6 +80,11 @@ def sanitize(env)
 
     protected
 
+    def skip?(rack_env_key)
+      return true if !@except.empty? && @except.any? { |matcher| rack_env_key[matcher] }
+      return true if !@only.empty? && @only.none? { |matcher| rack_env_key[matcher] }
+    end
```

Comment:
Nit: this should return `true` or `false`, not `true` or `nil`.

Reply:
Ok.